schema/patient.js

```diff
@@ -0,0 +1,29 @@
+const patientSchema = `
+enum triageLevel {
+  GREEN
+  YELLOW
+  RED
+}
+  extend type Query {
+    patient(id: Int!): Patient
+    patients: [Patient]
+  }
+  
+  type Patient {
+    gender: String,
+    age: Int,
+    runNumber: Int,
+    barcodeValue: Int!,
+    incidentId: Int!,
+    status: String,
+    triageLevel: triageLevel, 
+    notes: String,
+    transportTime: DateTime,
+    createdAt: DateTime,
+  }
```

Comment:
some fields need to be updated
```suggestion
  type Patient {
    id: ID!,
    gender: String,
    age: Int,
    runNumber: Int,
    barcodeValue: Int!,
    incidentId: Int!,
    status: String,
    triageCategory: Int,
    triageLevel: triageLevel, 
    notes: String,
    transportTime: DateTime,
    createdAt: DateTime,
    updatedAt: DateTime,
  }
```

Reply:
`status` should be changed to an enum with the following options: `ON_SITE`, `RELEASED`, `TRANSPORTED`

Reply:
`triageCategory` is a new field that can be an `Int` from 1 to 5